### PR TITLE
Contributors List: Usernames are involving, Code Cleanup, Rename and Footer Link

### PR DESCRIPTION
**Pull request by GabsEdits**
Some Changes since now it's a web page. Ready to Merge

### CODE REVIEW

File: get-all.sh [hunk 1]
```diff
@@ -6,8 +6,6 @@ echo
 
 owner="Vanilla-OS"
 
-# set -x
```
Comment (mirkobrombin) [inline get-all.sh:9]: I left that on purpose to easily enable/disable debugging

Comment (kbdharun) [reply get-all.sh]: Added it back.

**Review by kbdharun — APPROVED** [review]

**Comment by GabsEdits** [comment]
Ready to merge.

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
Tried fixing the merge conflicts but was unsuccessful, maybe this PR can be merged despite the conflict as it significantly modifies the existing file.

**Comment by GabsEdits** [comment]
> Tried fixing the merge conflicts but was unsuccessful, maybe this PR can be merged despite the conflict as it significantly modifies the existing file.

Same bug again.

**Comment by GabsEdits** [comment]
The Pull Request will be redone for the new Vue.js website.

**Comment by GabsEdits** [comment]
This pull request is getting closed as it will be replaced by #123

**Pull request closed without merging**